staging/src/slime.io/slime/modules/lazyload/controllers/process.go

```diff
@@ -94,6 +95,12 @@ func (r *ServicefenceReconciler) Refresh(req reconcile.Request, value map[string
 		return reconcile.Result{}, nil
 	}
 
+	log.Debugf("refresh with servicefence %s metricstatus old: %v, new: %v", req.NamespacedName, sf.Status.MetricStatus, value)
+	// skip refresh when metric result has not changed
+	if (len(sf.Status.MetricStatus) == 0 && len(value) == 0) ||
```

Comment:
`Equals` is easy to impl for `map[string]string`

Reply:
replace by `mapStrStrEqual`